.travis.yml

```diff
@@ -4,13 +4,15 @@ language: python
 
 jobs:
   include:
+    - { python: "3.10", env: "TOXENV=py310" }
+    - { python: "3.9", env: "TOXENV=py39" }
     - { python: "3.8", env: "TOXENV=py38" }
     - { python: "3.7", env: "TOXENV=py37" }
     - { python: "3.6", env: "TOXENV=py36" }
 
     - stage: Release
       if: tag IS present
-      python: "3.8"
+      python: "3.9"
```

Comment:
Kept this at 3.9 or should CI run against 3.10?

Reply:
Might as well keep this at the latest version, so 3.10

Reply:
```suggestion
      python: "3.10"
```